chart/compass/charts/kyma-environment-broker/templates/deployment.yaml

```diff
@@ -45,13 +55,23 @@ spec:
               value: "{{ .Values.broker.port }}"
             - name: APP_AUTH_USERNAME
               value: "{{ .Values.broker.username }}"
-            - name: APP_PROCESS_RUNTIME_ID
-              value: "{{ .Values.provisioner.processRuntimeID }}"
             - name: APP_AUTH_PASSWORD
               valueFrom:
                 secretKeyRef:
                   name: {{ include "kyma-env-broker.fullname" . }}
                   key: broker-password
+            - name: APP_PROCESS_RUNTIME_ID
+              value: "{{ .Values.provisioner.processRuntimeID }}"
+            - name: APP_DIRECTOR_NAMESPACE
+              value: "{{ .Release.Namespace }}"
+            - name: APP_DIRECTOR_DEFAULT_TENANT
+              value: "{{ .Values.global.defaultTenant }}"
```

Comment:
Do we need `APP_DIRECTOR_DEFAULT_TENANT` as a parameter?

Reply:
we don't, good catch